OWNERS

```diff
@@ -6,6 +6,7 @@ approvers:
   - qbarrand
   - yevgeny-shnaidman
   - ybettan
+  - enriquebelarte
```

Comment:
Could you please follow the alphabetical order here? Same for the `reviewers` section below. Thanks!

Reply:
Done